src/app/(mobile-ui)/[recipient]/error.tsx

```
@@ -0,0 +1,26 @@
+'use client'
+
+import { Button, Card } from '@/components/0_Bruddle'
+import { useEffect } from 'react'
+
+export default function Error({ error, reset }: { error: Error & { digest?: string }; reset: () => void }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)